salt/docker_apps/files/kingfisher_process.yaml

```diff
@@ -3,8 +3,6 @@ version: "2.2"
 x-shared: &shared
   image: "ghcr.io/open-contracting/kingfisher-process:latest"
   network_mode: "host"
-  env_file:
-    - env
   volumes:
     - /data:/data
```

Comment:
Should this also be /data/storage? (I haven't checked if it reads outside there.)

Reply:
I didn't check for the kingfisher-process Dockerfile yet. From a quick look, I see the /data directory is created in the Dockerfile. From the deploy repo I see https://github.com/open-contracting/deploy/blob/9298e0690da0190dd62221a84d6b2ff0d1c14689/pillar/registry.sls#L54 but we are not setting the `KINGFISHER_API_LOCAL_DIRECTORY` as in https://github.com/open-contracting/deploy/blob/9298e0690da0190dd62221a84d6b2ff0d1c14689/pillar/kingfisher.sls#L78, so I'm not sure how kingfisher-collect and kingfisher-process are connected in the registry server and therefore, for what the /data directory is used in kingfisher-process.

Reply:
`KINGFISHER_API_LOCAL_DIRECTORY` is only required for the old Process integration. It is not used by the new Process integration. In both cases, however, Process reads the JSON files written by Collect.

Reply:
So Kingfisher process will need to have access to /data/storage/kingfisher-collect? If so we can update KF to create that same directory in its Dockerfile and then update the volume here or leave it as it is now

Reply:
Yes, it needs that access. We might as well update its Dockerfile to make it follow the common template: https://ocp-software-handbook.readthedocs.io/en/latest/docker/index.html#templates

Reply:
Great to see that you made a template now!

Reply:
done in https://github.com/open-contracting/kingfisher-process/pull/344